tools/topology/sof-imx8ulp-btsco.m4

```diff
@@ -0,0 +1,82 @@
+#
+# Topology for i.MX8ULP board with wm8960 codec
```

Comment:
Please explain here what are the DAIs, what is the codec. Some information about the audio topology written in words is more than necessary. 

I bet no one knows what does btsco means.